scripts/globals/additional_effects.lua

```diff
@@ -75,19 +89,21 @@ xi.additionalEffect.statusAttack = function(addStatus, defender)
     return 0
 end
 
+-- Todo: swap from using element to damageType enum, so Excalibur etc. can happen.
 xi.additionalEffect.calcDamage = function(attacker, element, defender, damage)
     local params = {}
 
     params.bonusmab   = 0
-    params.includemab = false
+    params.includemab = false -- May possibly need to include mab on case by case basis, further tests needed
```

Comment:
Out of scope for this PR, but if this can get isolated to WS's, then I can nuke it into oblivion

Reply:
addBonusesAbility I think is the part using this atm. I could def rework add effects to check for and use it in another way in the event I had to, and right now zero of them do so if you want to murder that param this is prime time to do it and we'd just leave the comment behind reminding me to finish looking into what items need to check mab if any. 

Itss some testing I've not gotten around to on retail yet. I think I checked low lv (15-30) add effect melee weapons with moldavite earring and that was it, need to check more and also holy bolts. My suspicion is ranged items will use it since they use int/mnd already.